tools/ci_build/github/linux/docker/build_scripts/install-build-packages.sh

```diff
@@ -29,6 +29,7 @@ fi
 
 
 if [ "${PACKAGE_MANAGER}" == "yum" ]; then
+	yum -y update
```

Comment:
Why is it needed?  We have more than 10 docker files using this script. You will need to run a lot of pipelines to test this change. If it is not absolutely necessary, please avoid changing it.

Reply:
I added "yum versionlock cuda* libcudnn* libnccl*" in install-entrypoint.sh. We don't know what else will be needed if you do this.

Reply:
Because on broken pipeline cuda113 docker images build progress, step 26 use "yum -y update" and found the latest version of xz. But on step 40, yum install cannot find the latest version of xz-devel. I want to add "yum -y update" to control the two step under the same environment.
But it seems a mirror issue, and we cannot control which mirror to use, "yum -y update" connot help. Revert it.